document/core/appendix/algorithm.rst

```diff
@@ -51,6 +51,7 @@ the latter surrounding :ref:`structured control instructions <syntax-instr-contr
      end_types : list(val_type)
      height : nat
      unreachable : bool
+     catch_label : bool
```

Comment:
I don't think we need this flag, given that we already store the opcode.

Reply:
removed